packages/deploy/hardhat.config.ts

```diff
@@ -21,6 +21,11 @@ const importedPackages = {
   ],
```

Comment:
packages/deploy/deploy/500_marketplace/510_deploy_exchange.ts should use TRUSTED_FORWARDER_V2 instead of TRUSTED_FORWARDER